### PR TITLE
docs: update the roadmap with our priorities at Q4

### PR DESCRIPTION
**Pull request by mgechev**
Update the Angular roadmap.

### PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/roadmap.md [hunk 1]
```diff
@@ -6,88 +6,96 @@ The projects below are not associated with a particular Angular version. We'll r
...
-### Angular versioning & branching
+### Streamline releases with consolidated Angular versioning & branching
 
 We want to consolidate release management tooling between Angular's multiple GitHub repositories ([angular/angular](https://github.com/angular/angular), [angular/angular-cli](https://github.com/angular/angular-cli), and [angular/components](https://github.com/angular/components)). This effort will allow us to reuse infrastructure, unify and simplify processes, and improve our release process's reliability.
 
-## Future
+### Optimized build speed and bundle sizes with Angular CLI webpack 5
 
-### Refresh introductory documentation
+As part of the v11 release, we introduced an opt-in preview of webpack 5 in the Angular CLI. To ensure stability, we’ll continue iterating on the implementation to enable build speed and bundle size improvements.
```
Comment (santoshyadavdev) [inline aio/content/guide/roadmap.md:55]: Can we mention this is experimental and only for yarn?

File: aio/content/guide/roadmap.md [hunk 2]
```diff
@@ -6,88 +6,96 @@ The projects below are not associated with a particular Angular version. We'll r
...
+
+### Simplified Angular mental model with optional NgModules
 
 To simplify the Angular mental model and learning journey, we’ll be working on making NgModules optional. This work will allow developers to develop standalone components and implement an alternative API for declaring the component’s compilation scope.
 
 ### Ergonomic component level code-splitting APIs
 
 A common problem of web applications is their slow initial load time. A way to improve it is to apply more granular code-splitting on a component level. To encourage this practice, we’ll be working on more ergonomic code-splitting APIs.
-
-### Migration to ESLint
```
Comment (nartc) [inline aio/content/guide/roadmap.md:91]: Is this done? or not on the immediate Roadmap anymore?

Comment (derekkite) [reply aio/content/guide/roadmap.md:91]: https://github.com/angular/angular-cli/issues/13732#issuecomment-719724640

File: aio/content/guide/roadmap.md (hunk 1 above)
Comment (mgechev) [reply aio/content/guide/roadmap.md:55]: I think opt-in preview is enough of a disclaimer. What do you think?

Comment (santoshyadavdev) [reply aio/content/guide/roadmap.md:55]: Makes sense on experimental part, but many developers use npm and they may raise this as issue.

Comment (mgechev) [reply aio/content/guide/roadmap.md:55]: We have instructions on enabling the preview in the blog post. I'd suggest keeping this outside the scope of the roadmap.

Comment (santoshyadavdev) [reply aio/content/guide/roadmap.md:55]: Great than 😊

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**